libs/ledger-live-common/src/device/use-cases/appDataBackup/DesktopStorageProvider.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  StorageProvider,
+  AppStorageType,
+  AppStorageKey,
+  appStorageTypeGuard,
+  BackupAppDataError,
+} from "./types";
+
+/**
+ * The storage provider for LLD.
+ * Implements the StorageProvider interface.
+ */
+export class DesktopStorageProvider implements StorageProvider<AppStorageType> {
```

Comment:
[ask] why do you introduce this class with the usage of `global.localStorage` in `ledger-live-common` instead of `ledger-live-desktop`? This value isn't defined in `ledger-live-mobile`

Reply:
I'll move it to LLD folder. In a mobile context, the use case (`backupAppDataUseCaseDI`) should be initialised by `MobileStorageProvider` with mobile features.